charts/datadoc/values.schema.json

```diff
@@ -93,79 +49,40 @@
         }
       }
     },
-    "ressurser": {
-      "description": "Velg minimum mengde RAM og CPU som skal være tilgjengelig i tjenesten.",
+    "features": {
+      "description": "Feature toggles",
+      "type": "object",
+      "properties": {}
+    },
```

Comment:
What does this do? Is it copied from another helm chart?

Reply:
It was there previously, i think the git diff thinks it was added when the other code just moved